rules/Php80/Rector/Class_/StringableForToStringRector.php

```diff
@@ -106,4 +118,38 @@ public function refactor(Node $node): ?Node
 
         return $node;
     }
+
+    private function processNotStringType(ClassMethod $toStringClassMethod): void
+    {
+        $hasReturn = $this->betterNodeFinder->hasInstancesOfInFunctionLikeScoped($toStringClassMethod, Return_::class);
+
+        if (! $hasReturn) {
+            $lastKey = array_key_last((array) $toStringClassMethod->stmts);
```

Comment:
it seems cause downgrade error:

https://github.com/rectorphp/rector-src/runs/4505677100?check_suite_focus=true#step:14:80

`DowngradeArrayKeyFirstLastRector` seems needs update for casting condition.

Reply:
I created PR https://github.com/rectorphp/rector-src/pull/1475 for it. It next need a tweak in `DowngradeArrayKeyFirstLastRector` to adapt this condition.

Reply:
Thanks :+1: